lib/token.js

```diff
@@ -136,14 +170,13 @@ function handleOAuthResponse(sdk, oauthParams, res, urls) {
 
   return new Q()
   .then(function() {
-    if (res['error'] || res['error_description']) {
-      throw new OAuthError(res['error'], res['error_description']);
-    }
-
-    if (res.state !== oauthParams.state) {
-      throw new AuthSdkError('OAuth flow response state doesn\'t match request state');
+    validateResponse(res, oauthParams);
+    if (res['code']) {
+      tokenTypes = pkce.loadMeta(sdk).responseType;
+      return exchangeCodeForToken(sdk, oauthParams, res['code'], urls);
```

Comment:
Can you please add a comment here saying that the hybrid flow isn't supported? So if we somehow received a `token` or `id_token` in our response, we'd disregard it.

Reply:
now "code" is explicitly disallowed as a responseType when calling getToken() variants. This should prevent any hybrid action using this library